webapp/components/Notification/Notification.vue

```diff
@@ -1,40 +1,81 @@
 <template>
-  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
+  <ds-space :class="{ read: notificationData.read, notification: true }" margin-bottom="x-small">
     <client-only>
       <ds-space margin-bottom="x-small">
-        <user-teaser :user="from.author" :date-time="from.createdAt" />
+        <user-teaser :user="notificationData.triggerer" :date-time="notificationData.createdAt" />
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:16:50Z" title="Thursday, January 23rd 2020, 8:16:50 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,40 +1,90 @@
 <template>
-  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
+  <ds-space :class="{ read: notificationData.read, notification: true }" margin-bottom="x-small">
     <client-only>
       <ds-space margin-bottom="x-small">
-        <user-teaser :user="from.author" :date-time="from.createdAt" />
+        <user-teaser :user="notificationData.triggerer" :date-time="notificationData.createdAt" />
```


`triggerer` is not actually a word...
maybe `submitter`?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T10:59:02Z" title="Wednesday, February 5th 2020, 11:59:02 am +01:00">Feb 5, 2020</time>_

---


I have found the word `triggerer` in DeepL.
And `submitter` will not hit the meaning good enough in my eyes.